assets/src/dashboard/components/dialog/dialog.js

```diff
@@ -0,0 +1,85 @@
+/*
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * External dependencies
+ */
+import PropTypes from 'prop-types';
+import { ThemeProvider } from 'styled-components';
+
+/**
+ * Internal dependencies
+ */
+import {
+  Dialog as StyledDialog,
+  Button,
+  BUTTON_SIZES,
+  BUTTON_TYPES,
+  theme,
+} from '../../../design-system';
+
+const Dialog = ({
+  onClose,
+  closeText,
+  onConfirm,
+  confirmText,
+  actions,
+  children,
+  ...rest
+}) => {
+  return (
+    <ThemeProvider theme={{ ...theme }}>
```

Comment:
I don't think we need to recreate this object
```suggestion
    <ThemeProvider theme={theme}>
```

Reply:
🤦  nope! thanks!